packages/cli/src/decorators/Scopes.ts

```diff
@@ -0,0 +1,17 @@
+import type { Scope, ScopeOptions } from '@n8n/permissions';
+import type { ScopeMetadata } from './types';
+import { CONTROLLER_REQUIRED_SCOPES } from './constants';
+
+export const RequireGlobalScope = (scope: Scope | Scope[], scopeOptions?: ScopeOptions) => {
```

Comment:
There seems to be quite an overlap between `Authorized` and `RequireGlobalScope`. If we update all the remaining controllers that still have `@Authorized` on them, we could combine the two decorators, as well as the middlewares that use the metadata from these decorators. 
We don't need to do that now, but we should do it soon, so that we don't end up maintaining a 3rd authorization mechanism, while we are still trying to get rid of `middlewares/auth.ts`